src/azure-cli/azure/cli/command_modules/role/commands.py

```diff
@@ -99,6 +99,16 @@ def load_command_table(self, _):
         g.custom_command('credential list', 'list_application_credentials')
         g.custom_command('credential delete', 'delete_application_credential')
 
+    # Register federated-credential command group under both app and sp.
+    # We use federated-credential instead of federated-identity-credential or fic, in order to align with
```

Comment:
We still need to finalize our choice on

- `federated-credential`
- `federated-identity-credential`
- `fic`

Reply:
federated-credential is probably a good middle ground

Reply:
Done. Renamed.